src/content/docs/plugins/submission.md

```diff
@@ -1,4 +1,115 @@
+
+### Submission Steps 
+
+1. **Fork the Repository**: Go to the specific repository on [https://github.com/Vendicated/Vencord](https://github.com/Vendicated/Vencord) page and select “Fork”. 
+
+2. **Clone Your Fork** 
+```bash 
+git clone https://github.com/YOUR_USERNAME/Vencord.git 
+cd Vencord 
+``` 
+
+3. **Add Your Plugin** 
+- Make sure that your plugin folder has the correct name (for example ‘myAwesomePlugin’) 
+- Next, make a copy of your plugin folder and place it in the `Vencord/src/plugins/` directory. 
+
+4. **Create a Branch** 
+```bash 
+git checkout -b add-plugin/yourPluginName 
+``` 
+
+5. **Commit Your Plugin** 
+```bash 
+git add src/plugins/yourPluginName 
+git commit -m "feat(Plugin): YourPluginName. 
+``` 
+Note: Replace ‘YourPluginName’ with your own actual plugin name 
+
+6. **Push to Your Fork** 
+```bash 
+git push –u origin add-plugin/YourPluginName 
+``` 
+
+7. **Open a Pull Request** 
+- Navigate to the fork that you created earlier on Github 
+- Click "Pull request" 
+- Set base repository to `Vendicated/Vencord` and base branch to `dev` 
+- Title your PR "feat(Plugin): YourPluginName
+- In the plugin description, the author should give a clear explanation of what the plugin does. 
+
+### Review Process 
+
+- The Vencord maintainers will simply go through your submission. 
+- Reply to any feedback as soon as possible or to changes that were requested 
+- It needs to be understood that changes might be made, thus preparing for such changes needs to be in place. 
+
+### Common Pitfalls 
+
+- Please make sure that the folder name of the plugin is the same as the one used in the commit message and the PR title 
+- Ensure that you have only committed files only found within your plugin’s directory. 
+- Make sure that your plugin does not clash with something that is already implemented in Vencord
+
+## Updating Your Plugin 
+
+Another important action is that after the plugin has been merged you may require adjustments or correction works. Here's how to do that
+
+1. **Sync Your Fork** 
+Ensure your fork is up-to-date with the main Vencord repository: 
+```bash 
+git checkout dev 
+git fetch upstream 
+git merge upstream/dev 
+git push origin dev 
+``` 
+
+2. **Opening a New Branch for your Update** 
+```bash 
+git checkout -b update-plugin/yourPluginName-description 
+``` 
+Instead of “description”, you should write a short tag of the change (for example, “fix-nitro-loading”). 
+
+3. **Make Your Changes** 
+Modify your plugin files in the directory src/plugins/yourPluginName/ 
+
+4. **Commit Your Changes** 
+```bash 
+git add src/plugins/yourPluginName 
+git commit -m "YourPluginName: fix Nitro users can not access the plugin 
+``` 
+Associated with the commit change, a brief and informative message should be written by the author. 
+
+5. **Push Your Changes** 
+```bash 
+git push -u origin update-plugin/yourPluginName-description 
```

Comment:
Overly verbose? Maybe instruct users to learn git branching earlier

Reply:
Added a note to the end 😀